### ISSUE TITLE
This is an urgent problem  when db upgrade  ！！！

### ISSUE THREAD

**Issue by GJson** [pardom-zz/ActiveAndroid#485]
Look this page  https://github.com/pardom/ActiveAndroid/wiki/Schema-migrations

When I want to upgrade the database encountered this problem “duplicate column name: online (code 1): , while compiling: ALTER TABLE Conversation ADD COLUMN online INTEGER ”, although this total probability is very small。
  You can add some to solve the field when column repeat what?  but, sqlite isn't support T-Sql、SP etc, and the db upgrade is excuted  like 4.sql , , ,  What should I do ?

**Comment by zhakid** [comment]
do you check if the table have a field named online if not try to ALTER TABLE Conversation DROP COLUMN online and then ALTER TABLE Conversation ADD COLUMN online INTEGER.

**Comment by GJson** [comment]
Yes this is the train of thought, judgment first,then  if not is added ......
But how to write in the file 3.sql ？As far as I know sqlite isn't support T-Sql、SP、Temporary variable assignment etc,  so  I confused is how to write this judgment in the file 3.sql .
